website/docusaurus.config.js

```diff
@@ -41,37 +41,88 @@ module.exports = {
       style: 'dark',
       links: [
         {
-          title: 'Docs',
```

Comment:
Let's keep it as `Docs` itself for now. We are going to refactor this part later. Thanks!